ui/app/templates/components/secret-edit.hbs

```diff
@@ -26,17 +26,18 @@
 <Toolbar>
   {{#unless (and (eq mode 'show') isWriteWithoutRead)}}
     <ToolbarFilters>
-      <input
-        data-test-secret-json-toggle=true
-        id="json"
-        type="checkbox"
-        name="json"
-        class="switch is-rounded is-success is-small"
-        checked={{showAdvancedMode}}
-        onchange={{action "toggleAdvanced" value="target.checked"}}
-        disabled={{and (eq mode 'show') secretDataIsAdvanced}}
-      />
-      <label for="json" class="has-text-grey">JSON</label>
+      <Switch
+        data-test-secret-json-toggle="true"
+        @inputId="json"
```

Comment:
if you'd like, you can just pass `id` here as opposed to `inputId`. makes it a little more clear that it's just going to be used for CSS.

Reply:
Afaik passing the ID to the component puts it on the wrapper, and the point of the ID here is to link the input with its label for accessibility purposes